packages/dd-trace/test/appsec/sdk/index.spec.js

```diff
@@ -0,0 +1,204 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
+const agent = require('../../plugins/agent')
+const getPort = require('get-port')
+const axios = require('axios')
+const appsec = require('../../../src/appsec')
+const tracer = require('../../../../../index')
+const Config = require('../../../src/config')
+describe('Appsec SDK', () => {
+  describe('calls to external methods', () => {
+    let trackUserLoginSuccessEvent, trackUserLoginFailureEvent, trackCustomEvent
+    let appsecSdk
+    const tracer = {}
+
+    beforeEach(() => {
+      trackUserLoginSuccessEvent = sinon.stub()
+      trackUserLoginFailureEvent = sinon.stub()
+      trackCustomEvent = sinon.stub()
+      const AppsecSdk = proxyquire('../../../src/appsec/sdk', {
+        './track_event': { trackUserLoginSuccessEvent, trackUserLoginFailureEvent, trackCustomEvent }
+      })
+      appsecSdk = new AppsecSdk(tracer)
+    })
+
+    it('trackUserLoginSuccessEvent should cal to track_event trackUserLoginSuccessEvent', () => {
+      const user = { id: 'user_id' }
+      const metadata = { key: 'value' }
+      appsecSdk.trackUserLoginSuccessEvent(user, metadata)
+      expect(trackUserLoginSuccessEvent).to.be.calledOnceWith(tracer, user, metadata)
+    })
+
+    it('trackUserLoginFailureEvent should cal to track_event trackUserLoginFailureEvent', () => {
+      const user = { id: 'user_id' }
+      const metadata = { key: 'value' }
+      appsecSdk.trackUserLoginFailureEvent(user, metadata)
+      expect(trackUserLoginFailureEvent).to.be.calledOnceWith(tracer, user, metadata)
+    })
+
+    it('trackCustomEvent should cal to track_event trackCustomEvent', () => {
+      const metadata = { key: 'value' }
+      appsecSdk.trackCustomEvent('event_name', metadata)
+      expect(trackCustomEvent).to.be.calledOnceWith(tracer, 'event_name', metadata)
+    })
+  })
+
+  describe('in request', () => {
+    let http
+    let controller
+    let appListener
+    let port
+    function listener (req, res) {
+      if (controller) {
+        controller(req, res)
+      }
+    }
+    before(async () => {
+    })
+
+    before(async () => {
+      port = await getPort()
+      await agent.load('http')
+      http = require('http')
+    })
+
+    before(done => {
+      const server = new http.Server(listener)
+      appListener = server
+        .listen(port, 'localhost', () => done())
+    })
+
+    after(() => {
+      appListener.close()
+      return agent.close({ ritmReset: false })
+    })
+
+    beforeEach(() => {
+      appsec.enable(new Config({
+        appsec: {
+          enabled: true
+        }
+      }))
+    })
```

Comment:
I think we can remove that, since all tests passes without it, and actually this part of the SDK should still work without appsec enabled remember we decided that on slack ?

Reply:
You're right, the SDK does not depend on appsec.enabled